grails-app/services/com/unifina/service/StreamService.groovy

```diff
@@ -170,7 +169,7 @@ class StreamService {
 			}
 
 			int partition = partitioner.partition(stream, null)
-			StreamMessageV30 msg = new StreamMessageV30(stream.id, partition, date.time, sequenceNumber, publisherId,
+			StreamMessageV30 msg = new StreamMessageV30(stream.id, partition, date.time, sequenceNumber, publisherId, "",
```

Comment:
Let's generate a random `msgChainId` before the `for` loop using `IdGenerator.getShort()` and use it here.